### ISSUE TITLE
QueryEvent doesn't respect only_schemas/ignore_schemas as row events

### ISSUE THREAD

**Issue by XUWeijiang** [noplay/python-mysql-replication#248]
Hi, guys.
First thank you for your great work.

I found that QueryEvent that will create/alter tables in other databases still returns even I set only_schemas. From source code, I can see that only_schemas/ignore_schemas are ignored in QueryEvent.

Expectation:
QueryEvent will respect the filtering arguments passed to BinLogStreamReader.

**Comment by baloo** [comment]
`CREATE TABLE` and `DROP TABLE` are only available as `QueryEvent`. Sadly there is no schema/table those events and we can filter them only by parsing sql. This is not something I intend to implement.
Sorry but this is a wontfix for me.

An alternative may be to only read `WriteRowsEvent`, `UpdateRowsEvent` and `DeleteRowsEvent` in `only_events` (see api of `BinLogStreamReader`)

**Comment by XUWeijiang** [comment]
Hi Baloo
I don't understand, but I can get schema name from QueryEvent. The field looks correct for me. In fact, I used QueryEvent.schema to filter the databases in my code. Can you help to confirm?

**Comment by baloo** [comment]
Yes, well you're right. I mixed up `only_schema` and `only_tables` here
